chevah/brink
# [#3331] Update buildbot try --wait to work outside of VPN.

Scope
=====

This fix the build output not being present in --wait after moving outside of VPN

Changes
=======

Update buildbot try to use the PB (Twisted RPC) for getting the remote logs.

In the past I used a simple hack to get logs over HTTP, but we now have HTTPS which is protected.
The right way to do is to use buildbot try all the way.

This is moslty about buildbot changes which are here https://github.com/chevah/buildbot/commit/ea0c16f78a9f4ade5dc78763a81fd84786fc1895 and in the next commit

brink was cleaned of methods which are no longer used.

Now you also get the URL to the build

Buildmaster was updated to support getting the build number together with the builder name.

brink was already released

How to test
=========

reviewers: @alibotean @dumol @brunogola

Check that changes are ok

From this branch run

```
# Get latest buildbot
paver deps

# Run a complete build on a slave
paver test_remote ubuntu-1404 --wait

# Run a specific test
paver test_remote ubuntu-1404 system.test_filesystem:TestBrinkFilesystem --wait
```

once this is merged, we will need to update all other projects
